askama_derive/src/parser.rs

```diff
@@ -290,6 +291,38 @@ named!(arguments<Input, Vec<Expr>>, do_parse!(
     (args.unwrap_or_default())
 ));
 
+named!(macro_arguments<Input, &str>, alt!(
+    delimited!(char!('('), nested_parenthesis, char!(')')) |
+    map!(delimited!(char!('('), take_until!(");"), tag!(");")),
```

Comment:
I don't want this `);` pattern in there, I don't think it fits with the rest of the design since it doesn't make sense to me that a template expression ends in `;`.

Reply:
and something less obvious like `))` could I get into?

Reply:
ok, without special end